plugins/modules/web_infrastructure/django_manage.py

```diff
@@ -288,9 +290,9 @@ def main():
 
     _ensure_virtualenv(module)
 
-    cmd = ["./manage.py", command]
+    cmd = ["./manage.py"] + command_split
 
-    if command in noinput_commands:
+    if command in noinput_commands and '--noinput' not in command_split:
```

Comment:
Should it be
```suggestion
    if command[0] in noinput_commands and '--noinput' not in command_split:
```
? (Or can options also be before it, and we need something like `any(cmd in noinput_commands for cmd in command)`?

Reply:
Errr..... no. See lines 277-278. The expanded array with all arguments is in `command_split` whilst `command` already contains only the first element. ;-)

Reply:
And writing that, I reckon I could refactor all these variables to clearer names ;-)

Reply:
Ah, right. I guess it should be `... command_split[0] in noinput_commands ...` then :) (Or one of the `command_split` parts?)

Reply:
I changed the variable `command` to `command_bin`, and its value ***is*** `command_split[0]`, so from where I am standing, it looks like the code is already doing what you're saying.

Reply:
Ah, I forgot about that last part when writing the previous comment! Everything's fine now :)